TurkishFilter/sections/whitelist.txt

```diff
@@ -1,4 +1,6 @@
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/71125
+@@||platform.twitter.com^$domain=tribundergi.com
```

Comment:
Please remove domain from the common rule which starts with `||platform.twitter.com/widgets.js$domain=` instead of the exception.